Makefile

```diff
@@ -110,7 +110,7 @@ help: ## Display this help.
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	$(CONTROLLER_GEN) rbac:roleName=manager-role crd$(CRDDESC_OVERRIDE) webhook paths="./..." output:crd:artifacts:config=config/crd/bases && \
-	rm -f api/bases/* && cp -a config/crd/bases api/
+	rm -r api/bases/ && cp -a config/crd/bases api/
```

Comment:
was that one required by the precommit test?

Reply:
Not really. Updating. Thanks for catch.